cmd/habitat/main.go

```diff
@@ -61,13 +75,29 @@ func main() {
 	go handleInterupt(ProcessManager)
 
 	// Create community manager
-	var err error
 	CommunityManager, err = community.NewManager(communityDir, ProcessManager, &reverseProxy.Rules, p2pNode.Host())
 	if err != nil {
 		log.Fatal().Msgf("unable to start community manager: %s", err)
 	}
 
-	listen()
+	apiURL, err := url.Parse("http://0.0.0.0:2040")
+	if err != nil {
+		log.Fatal().Msgf("unable to get url for Habitat API: %s", err)
+	}
+	reverseProxy.Rules.Add("habitat-api", &proxy.RedirectRule{
+		Matcher:         "/habitat",
+		ForwardLocation: apiURL,
+	})
+
+	instance := &Instance{
```

Comment:
i think i missed this somewhere - what is instance supposed to be? an instance of habitat node?